src/AccessibilityInsights.SharedUx/Dialogs/HotkeyGrabDialog.xaml.cs

```diff
@@ -50,6 +40,7 @@ private void Window_KeyDown(object sender, KeyEventArgs e)
             switch (tempKey)
             {
                 case Key.Escape:
+                case Key.Tab:
```

Comment:
Should we exclude shift as well? I think it might be worthwhile - VS IDE seems to exclude it and it might allow users to shift-tab without us capturing